composer.json

```diff
@@ -45,10 +45,10 @@
     "require-dev": {
         "doctrine/coding-standard": "^14",
         "doctrine/orm": "^3.4.4",
-        "phpstan/phpstan": "2.1.1",
+        "phpstan/phpstan": "2.1.13",
         "phpstan/phpstan-phpunit": "2.0.3",
         "phpstan/phpstan-strict-rules": "^2",
-        "phpstan/phpstan-symfony": "^2.0",
+        "phpstan/phpstan-symfony": "2.0.9",
```

Comment:
What happened here? Caret version is preferrable

Reply:
With caret version the phpstan-symfony was picking only 2.0.3 and 2.0.9 fix a class.NotFound 
https://github.com/phpstan/phpstan-symfony/issues/464

Reply:
Done